rebar.config

```diff
@@ -21,7 +21,7 @@
 
 {erl_opts, [
     {lager_extra_sinks, ['__lager_test_sink']},
-    {platform_define, "^(19|20|21|22)", test_statem},
+    {platform_define, "^(19|20|21|22|23|24)", test_statem},
```

Comment:
Probably not the best approach. Suggestions are welcome.

Reply:
Arguably we'd remove 19 and 20 from this list since we only support the last two stable releases.

Possibly we can also just kill this check entirely now.